.tools/test/sqs_lambda_to_batch_fargate/lambda/export_logs.py

```diff
@@ -60,10 +58,16 @@ def get_and_put_logs():
     )
     file_identifier = str(random.randint(10**7, 10**8 - 1))
```

Comment:
And we're still unsure of any more identifiable information we could put in the event to get a human-readable consistent name?

Reply:
The only other option I considered was getting the ID of the Batch job that completed so that a user could potentially link the Lambda run with the Batch job ID. It would be a much longer string. Does this sound better?

Reply:
Yes, it's at least deterministic.

Reply:
Changes pushed.